tests/Eccube/Tests/Web/Admin/Setting/System/SecurityControllerTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/*
+ * This file is part of EC-CUBE
+ *
+ * Copyright(c) 2000-2015 LOCKON CO.,LTD. All Rights Reserved.
+ *
+ * http://www.lockon.co.jp/
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ */
+
+
+namespace Eccube\Tests\Web\Admin\Setting\Shop;
+
+use Eccube\Tests\Web\Admin\AbstractAdminWebTestCase;
+use Symfony\Component\Yaml\Yaml;
+
+class SecurityControllerTest extends AbstractAdminWebTestCase
+{
+    protected $configFile;
+    protected $configFileBackup;
+
+    protected $pathFile;
+    protected $pathFileBackup;
+    protected $ipTest = '192.168.1.100';
+    public function setUp()
+    {
+        parent::setUp(); // TODO: Change the autogenerated stub
+        if (!$this->backupConfig() || !$this->backupPath()) {
+            $this->markTestSkipped('Backup fail!!!');
+        }
+//        $config = Yaml::parse(file_get_contents($this->configFile));
+//        $config['admin_allow_host'] = array($this->ipTest);
+//        file_put_contents($this->configFile, Yaml::dump($config));
+    }
+
+    public function tearDown()
+    {
+        $this->rollbackConfig();
+        $this->rollbackPath();
+        parent::tearDown(); // TODO: Change the autogenerated stub
+    }
+
+    protected function backupConfig()
```

Comment:
Proposal use of [vfsStream](http://vfs.bovigo.org/).
see also, https://github.com/EC-CUBE/ec-cube/blob/91ca6d9fdc4c60d68562284a5785d052b637933c/tests/Eccube/Tests/Web/UserDataControllerTest.php